src/worknet/Node/PluginHandler.cs

```diff
@@ -0,0 +1,46 @@
+using System.Reflection;
+using Neo.Plugins;
+using McMaster.Extensions.CommandLineUtils;
+
+class PluginHandler
+{
+    public static void LoadPlugins(string directory, IConsole console)
```

Comment:
Since IConsole is a type specific to the command line library WorkNet uses, we should pass this as a TextWriter instead. Maybe we make it optional instead? Both of these will make it easier to update platform logic and eliminate the need for this class entirely.